### PR TITLE
containerd: disable memory update test on ppc64le

### LINKED ISSUE #3580: containerd: memory update test fails on ppc64le
The recently introduced container memory update test fails on ppc64le:

```
INFO: Test container memory update without virtio-mem
Image is up to date for sha256:ae9f60da61973d444fd11c7aa1ab5966c33b1b7e4773c7cd922afa509d3489c8
82387895ef6ac29ec215d7c77d09775d5f2849eb7a76122b4349cdc70ae0f546
2021/05/28 07:56:19 (0xc000416000) (0xc00041e000) Create stream
2021/05/28 07:56:19 (0xc000416000) (0xc00041e000) Stream added, broadcasting: 1
2021/05/28 07:56:19 (0xc000416000) Reply frame received for 1
2021/05/28 07:56:19 (0xc000416000) (0xc0002b8640) Create stream
2021/05/28 07:56:19 (0xc000416000) (0xc0002b8640) Stream added, broadcasting: 3
2021/05/28 07:56:19 (0xc000416000) Reply frame received for 3
2021/05/28 07:56:19 (0xc000416000) (0xc0002720a0) Create stream
2021/05/28 07:56:19 (0xc000416000) (0xc0002720a0) Stream added, broadcasting: 5
2021/05/28 07:56:19 (0xc000416000) Reply frame received for 5
2021/05/28 07:56:19 (0xc000416000) Data frame received for 3
2021/05/28 07:56:19 (0xc0002b8640) (3) Data frame handling
2021/05/28 07:56:19 (0xc0002b8640) (3) Data frame sent
2021/05/28 07:56:19 (0xc000416000) Data frame received for 5
2021/05/28 07:56:19 (0xc0002720a0) (5) Data frame handling
2021/05/28 07:56:19 (0xc000416000) Data frame received for 3
2021/05/28 07:56:19 (0xc0002b8640) (3) Data frame handling
2021/05/28 07:56:19 (0xc000416000) Data frame received for 1
2021/05/28 07:56:19 (0xc00041e000) (1) Data frame handling
2021/05/28 07:56:19 (0xc00041e000) (1) Data frame sent
2021/05/28 07:56:19 (0xc000416000) (0xc00041e000) Stream removed, broadcasting: 1
2021/05/28 07:56:19 (0xc000416000) (0xc00041e000) Stream removed, broadcasting: 1
2021/05/28 07:56:19 (0xc000416000) (0xc0002b8640) Stream removed, broadcasting: 3
2021/05/28 07:56:19 (0xc000416000) (0xc0002720a0) Stream removed, broadcasting: 5
time="2021-05-28T07:56:19-04:00" level=fatal msg="Updating container resources for \"82387895ef6ac29ec215d7c77d09775d5f2849eb7a76122b4349cdc70ae0f546\" failed: rpc error: code = Unknown desc = failed to update resources: failed to update resources: File exists (os error 17): unknown"
```

### PR DESCRIPTION
**Pull request by MatthieuSarter**
Fixes #3580

Signed-off-by: Matthieu Sarter <matthieu.sarter@ibm.com>

### CODE REVIEW

**Review by jodh-intel — APPROVED** [review]
Thanks @MatthieuSarter.

lgtm

File: integration/containerd/cri/integration-tests.sh [hunk 1]
```diff
@@ -263,9 +263,9 @@ TestKilledVmmCleanup() {
 }
 
 TestContainerMemoryUpdate() {
-	if [[ "${KATA_HYPERVISOR}" != "qemu" ]]; then
+	if [[ "${KATA_HYPERVISOR}" != "qemu" ]] || [[ "${ARCH}" == "ppc64le" ]]; then
 		return
-	fi
+	fi:wq
```
Comment (bpradipt) [inline integration/containerd/cri/integration-tests.sh:268]: @MatthieuSarter typo ? ^^

Comment (MatthieuSarter) [reply integration/containerd/cri/integration-tests.sh]: Yep... Fixed.

**Review by bpradipt — APPROVED** [review]
/lgtm

**Pull request merged**